### ISSUE TITLE
"Public Access" access group is not listed in the result

### ISSUE THREAD

**Issue by khayama** [IBM-Cloud/ibm-cloud-cli-release#137]
I can see any "Public Access" access group is not listed in the result

```
ibmcloud iam access-groups
```

However, I can see it using the following command

```
export ACCOUNT_ID=xxx
export ACCESS_TOKEN=$(ibmcloud iam oauth-tokens --output JSON | jq -r '.iam_token')

curl -X GET "https://iam.cloud.ibm.com/v2/groups?account_id=$ACCOUNT_ID" -H "Authorization: $ACCESS_TOKEN" -H 'Content-Type: application/json' | jq -r
```

Please fix it, thanks.

**Comment by archerzz** [comment]
@khayama This issue has been fixed in version `1.3.0`. Please upgrade to the latest version. Thanks.

**Comment by khayama** [comment]
I have confirmed it works at version `1.3.0`, thanks for your comment.
